modules/db/src/main/QueryBuilderExt.scala

```diff
@@ -27,10 +27,9 @@ trait QueryBuilderExt { self: dsl =>
     // like one, but with stopOnError defaulting to false
     def uno[A: BSONDocumentReader]: Fu[Option[A]] = uno[A](ReadPreference.primary)
 
-    def uno[A: BSONDocumentReader](readPreference: ReadPreference): Fu[Option[A]] =
-      b.copy(options = b.options.batchSize(1))
-        .cursor[A](readPreference = readPreference)
-        .collect[Iterable](1, stopOnError = false)
-        .map(_.headOption)
+    def uno[A: BSONDocumentReader](readPreference: ReadPreference): Fu[Option[A]] = b.copy(options = b.options.batchSize(1))
+      .cursor[A](readPreference = readPreference)
+      .collect[Iterable](1, Cursor.ContOnError[Iterable[A]]())
```

Comment:
@ornicar Not sure what's the difference against `headOption` or `one` ... maybe legacy?